pos_gift_ticket/static/src/js/main.js

```diff
@@ -5,44 +5,58 @@
     The licence is in the file __openerp__.py
 */
 
+odoo.define('pos_gift_ticket.gift_print_widget', function(require) {
+    "use strict";
 
-openerp.pos_gift_ticket = function (instance) {
-    var _t = instance.web._t,
-        _lt = instance.web._lt;
-    var QWeb = instance.web.qweb;
-
-    instance.point_of_sale.ReceiptScreenWidget.include({
-        show: function(){
-            this._super()
-            var self = this;
-            var print_gift_ticket_button = this.add_action_button({
-                    label: _t('Gift Ticket'),
-                    icon: '/point_of_sale/static/src/img/icons/png48/printer.png',
-                    click: function(){
-                        self.print_gift();
-                    },
-                });
-        },
+    var core = require('web.core');
+    var QWeb = core.qweb;
+    var _t = core._t;
+
+    var HeaderButtonWidget = require('point_of_sale.chrome').HeaderButtonWidget;
+
+    var GiftPrintWidget = HeaderButtonWidget.extend({
         refresh_gift_ticket: function(){
             var order = this.pos.get('selectedOrder');
             $('.pos-receipt-container', this.$el).html(QWeb.render('PosGiftTicket',{
```

Comment:
I've edited line 20 like this:
"$('.pos-receipt-container').html(QWeb.render('PosGiftTicket',{"
to make it work properly.